test/util/e2e.go

```diff
@@ -65,11 +65,19 @@ func CreateVisibilityClient(user string) visibilityv1alpha1.VisibilityV1alpha1In
 	return visibilityClient
 }
 
-func KueueReadyForTesting(ctx context.Context, client client.Client) {
+func KueueReadyForTesting(ctx context.Context, c client.Client) {
```

Comment:
Can we delete this function if we are waiting for the "webooks setup before ready"?

Reply:
no, this is running in the test binary

Reply:
but should not the test binary wait until Kueue is healthy / ready, rather than trying to periodically create the items? If we still periodically need to create items I don't seem much gain of delaying the healthy status.

Reply:
It's just another way of checking the same thing.

I theory you can check if all the kueue-controller-manager pods are ready instead.

Reply:
I see, but this involves creating / deleting objects, something which users may not want to do in prod. I think it is better if e2e tests correspond better to production use. Also, the creation / deletion takes more time some may cover some user issues. If we go with this approach I would suggest to wait as for ready deployment, this can be done as here: 
```
kubectl wait --for=condition=available --timeout=3m deployment/kueue-controller-manager -n kueue-system
```